src/main/java/uk/gov/digital/ho/hocs/queue/ukvi/UKVIComplaintData.java

```diff
@@ -42,22 +45,31 @@ public String getComplaintType() {
     }
 
     @Override
-    public ComplaintCorrespondent getComplaintCorrespondent() {
-        String applicantType = ctx.read(APPLICANT_TYPE);
-        ComplaintCorrespondent correspondent;
+    public ArrayList<ComplaintCorrespondent> getComplaintCorrespondent() {
 
-        if (applicantType.equals("APPLICANT")) {
-            correspondent = new ComplaintCorrespondent(ctx.read(APPLICANT_APPLICANT_NAME));
-            optionalString(ctx, APPLICANT_APPLICANT_EMAIL).ifPresent(correspondent::setEmail);
-            optionalString(ctx, APPLICANT_APPLICANT_PHONE).ifPresent(correspondent::setTelephone);
+        ArrayList<ComplaintCorrespondent> correspondents = new ArrayList<>();
 
-        } else if (applicantType.equals("AGENT")) {
-            correspondent = new ComplaintCorrespondent(ctx.read(AGENT_APPLICANT_NAME));
-
-        } else {
-            throw new IllegalStateException("APPLICANT_TYPE Unknown : " + applicantType);
+        try {
+            String applicantType = ctx.read(APPLICANT_TYPE);
+            if (applicantType.equals("APPLICANT")) {
+                ComplaintCorrespondent applicantCorrespondent = new ComplaintCorrespondent(ctx.read(APPLICANT_APPLICANT_NAME), "COMPLAINANT");
+                optionalString(ctx, APPLICANT_APPLICANT_EMAIL).ifPresent(applicantCorrespondent::setEmail);
+                optionalString(ctx, APPLICANT_APPLICANT_PHONE).ifPresent(applicantCorrespondent::setTelephone);
+                correspondents.add(applicantCorrespondent);
+            } else if (applicantType.equals("AGENT")) {
+                ComplaintCorrespondent applicantCorrespondent = new ComplaintCorrespondent(ctx.read(AGENT_APPLICANT_NAME), "COMPLAINANT");
+                ComplaintCorrespondent agentCorrespondent = new ComplaintCorrespondent(ctx.read(AGENT_AGENT_NAME), "TPR");
```

Comment:
What is TPR sorry?

Reply:
Third Party Representative - I've taken this value from the correspondent_type table. Going to talk with @MarkWrightBjss about the best way to handle these values next week.